examples/topics/geo/tile_xyzservices.py

```diff
@@ -1,3 +1,16 @@
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map. The 'tile_xyz' service provides map tiles, 
```

Comment:
This needs a little adjustment. The package that provides the tiles services is [`xyzservivces`](https://github.com/geopandas/xyzservices) (and not 'tile_xyz') we might want to link to the `xyzservices` project here. Also it's not a hard and fast rule, but I've been trying to keep the line length of docstring text for examples closer to 80-100 characters wide, so I'd suggest shortening the lines somewhat.

Reply:
Thanks a lot for the feedback. @bryevdv.
I noticed the code regarding the plot is showing error so can i just create a PR or i need to create it as an issue first before PR